commands/volumes/brickutils.go

```diff
@@ -0,0 +1,116 @@
+package volumecommands
+
+import (
+	"os/exec"
+	"syscall"
+	"time"
+
+	"github.com/gluster/glusterd2/brick"
+	"github.com/gluster/glusterd2/daemon"
+	"github.com/gluster/glusterd2/peer"
+	"github.com/gluster/glusterd2/utils"
+
+	"github.com/pborman/uuid"
+)
+
+// BrickStartMaxRetries represents maximum no. of attempts that will be made
+// to start brick processes in case of port clashes.
+const BrickStartMaxRetries = 3
+
+// Until https://review.gluster.org/#/c/16200/ gets into a release.
+// And this is fully safe too as no other well-known errno exists after 132
+const anotherEADDRINUSE = syscall.Errno(0x9E) // 158
+
+func errorContainsErrno(err error, errno syscall.Errno) bool {
+	if exiterr, ok := err.(*exec.ExitError); ok {
```

Comment:
This piece of code is just moved here from a different file.

It was in-lined when I wrote the code initially. During the PR review, there was suggestion to move it out to improve readability (it was a lot of indentation and nested ifs). Specifically, [here](https://github.com/gluster/glusterd2/commit/76fd6745404b8b0eec98105cd339ced1c088c2b8) is the commit where I made it not inline.